control/control_test.go

```diff
@@ -1606,10 +1551,25 @@ func (l *listenToPluginSubscriptionEvent) HandleGomitEvent(e gomit.Event) {
 		}).Info("Unhandled Event")
 	}
 }
+
+func (l *listenToPluginLoadEvent) HandleGomitEvent(e gomit.Event) {
+	switch v := e.Body.(type) {
+	case *control_event.LoadPluginEvent:
+		l.plugin.EventNamespace = v.Namespace()
+		l.done <- struct{}{}
+	default:
+		controlLogger.WithFields(log.Fields{
+			"event:": v.Namespace(),
+			"_block": "HandleGomit",
+		}).Info("Unhandled Event")
+	}
+}
+
+/*
```

Comment:
I agree except the test probably needs to be modified to work with the new changes. We want to test metric subscription rollover.